test/docker-compose.yml

```diff
@@ -0,0 +1,69 @@
+# NB: This compose file is intended to be executed via the Makefile in the root
+# of this repository. See the relative paths to the "build" directory in the
+# volume definitions below.
+version: '3'
+
+services:
+  node0:
+    container_name: node0
+    image: "tendermint/localnode"
+    ports:
+      - "26656-26657:26656-26657"
+    environment:
+      - ID=0
+      - LOG=${LOG:-tendermint.log}
+    volumes:
+      - ../build:/tendermint:Z
+    networks:
+      localnet:
+        ipv4_address: 192.167.10.2
```

Comment:
To be compliant with RFC1918 (regarding non publicly routable addresses), please use a 192.168.x.x IP range

Reply:
This is taken as-is from the Tendermint Core Docker Compose file: https://github.com/tendermint/tendermint/blob/a9feb1c023e172b542c972605311af83b777855b/docker-compose.yml

I'll try change it here, but it should be changed in the consensus engine too. I have a suspicion it was set to this range for a reason, but it's not documented anywhere as to why this was chosen.